config/webpacker.yml

```diff
@@ -51,10 +50,6 @@ default: &default
 development:
   <<: *default
   compile: true
-  extract_css: true
```

Comment:
So I had to turn this on previously because:

a) having the CSS loaded in via Javascript in development, but not production just makes me very nervous. Probably because I'm being freaked out by all the modern day FE pipeline stuff.
b) I was validated in me being weirded out! The PDF renderer can't handle this approach to loading CSS, so in development, the PDFs lose their CSS

And I can see that webpacker still behaves like this if we remove this line: the PDFs don't have styling in development mode.

I have a proposed solution, based off of skimming this very detailed discussion: https://github.com/rails/webpacker/issues/2605

I created a new file in `app/javascript/packs/application.scss` which just included the following line:

`@import "stylesheets/application";`

I then turned `extract_css` back on for development. This appears to have fixed everything for me locally.